mofacts/client/views/experiment/card.js

```diff
@@ -2666,12 +2664,30 @@ async function resumeFromComponentState() {
 
   await updateExperimentState(newExperimentState, 'card.resumeFromComponentState');
 
+  getFeebackParameters(curTdfUnit);
```

Comment:
I think when you rolled back some it took out the spelling correction of feedback here?